### PR TITLE
Mobile publish fixes

### PR DESCRIPTION
**Pull request by EnzoVezzaro**
Closes [Mobile publish flow (layout)#1075](https://github.com/oceanprotocol/market/issues/1075).

Changes proposed in this PR:

- check all publish screens on mobile
- adapt problematic component layouts as needed (not really needed, everything looks ok)
- check all form navigation for user surprises (e.g. using Continue scrolls to top of viewport) - didn't find any issue on the forms

### CODE REVIEW

File: src/components/Publish/Pricing/Fees.module.css [hunk 1]
```diff
@@ -18,3 +18,9 @@
 .fees input {
   max-width: 5rem;
 }
+
+@media screen and (max-width: 41rem) {
```
Comment (kremalicious) [inline src/components/Publish/Pricing/Fees.module.css:22]: we do mobile first in our CSS codebase so only work with `min-width` and adapt elements as you scale up viewports. So the default for all elements is their mobile view and as you scale up the viewport they get adapted if needed

**Comment by kremalicious** [comment]
would suggest looking at the NFT & Datatoken form fields which to me look like they should be stacked by default and then get left/right placement as viewport gets bigger.

File: src/components/Publish/Pricing/Fees.module.css (hunk 1 above)
Comment (EnzoVezzaro) [reply src/components/Publish/Pricing/Fees.module.css]: Sorry for this weird correction. For some reason my local github wasn't able to push to the correct branch. It got stuck on the re-named old branch.

**Comment by EnzoVezzaro** [comment]
Correction for mobile merged to PR

**Comment by EnzoVezzaro** [comment]
@kremalicious thank you, you're right. Fixed in the last commits

**Comment by kremalicious** [comment]
when switching between _Metadata_ & _Access_, the Data NFT and Datatoken fields should look the same. Right now the NFT is stacked, but Datatoken is still left/right placement on mobile. Although Datatoken field kinda fits better, being consistent is more important here.

Also, would just left-align image/text on mobile, centering that NFT image looks unbalanced when input label above and content below is left-aligned.

**Comment by kremalicious** [comment]
Found 2 more things we can improve:

Using really narrow viewport here but still seems 2 fee fields at bottom would still fit beside each other, we could also make them slightly smaller by default to gain a bit more space. On same screen the tokens in the pool box are misaligned, should both be left aligned when they are stacked like that, otherwise looks chaotic and unbalanced in current state.

Screen Shot 2022-02-15 at 17 11 19

On submission screen, number & title should always be on one line, and the transaction label should be then in new line, left aligned with the edge of the text beneath it

Screen Shot 2022-02-15 at 17 12 17

**Comment by EnzoVezzaro** [comment]
pricing and submit tabs fixed for small viewports 🤙🏽

**Review by kremalicious — APPROVED** [review]

**Pull request merged**